### PR TITLE
Stimpak.cast feature

### PR DESCRIPTION
**Pull request by warang580**

### CODE REVIEW

File: source/spec/stimpak/stimpak.generate.cast.spec.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import Stimpak from "../../lib/stimpak/stimpak.js";
+
+describe("stimpak.generate() (casted answers)", () => {
+	let stimpak;
+
+	let callbackA = answer => parseInt(answer);
```
Comment (dcrockwell) [inline source/spec/stimpak/stimpak.generate.cast.spec.js:6]: Fix whitespacing to be tabs-only

File: source/spec/stimpak/stimpak.casts.spec.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import Stimpak from "../../lib/stimpak/stimpak.js";
+
+describe("stimpak.casts()", () => {
+	let stimpak;
+  let callbackA = answer => parseInt(answer);
```
Comment (dcrockwell) [inline source/spec/stimpak/stimpak.casts.spec.js]: whitespacing: tabs only

**Comment by dcrockwell** [comment]
Very nice, I left a couple of formatting consideration notes

**Comment by dcrockwell** [comment]
There's an extra scenario as well which needs a test to cover off on it (note there is no `.prompt`):

instead of this, you can do:

``` javascript
stimpak
.destination(process.cwd())
.answers({
    "casted": "5",
    "untouched": "5"
})
.cast(answer => parseInt(answer))
.generate(error => {
    stimpak.answers().casted.should.eql(5);
    stimpak.answers().untouched.should.eql("5");
});
```

**Comment by warang580** [comment]
I've added the test and cleaned the indentation with tabs only.

Just one question, what is the expected behaviour when doing this : 

``` javascript
stimpak
  .cast(answer => parseInt(answer))
  .prompt(promptOne)
  .prompt(promptTwo)
```

Are promptTwo answers (and also following prompts answers) also casted ? Or is it just the first .prompt() after .cast() ?

**Comment by dcrockwell** [comment]
> Just one question, what is the expected behaviour when doing this :

For now, I imagine the .cast working on all subsequent prompts, but in the future we could make it per-prompt

**Pull request merged**